.ci/bump-maven-pom-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+POM_FILE=maven-project/pom.xml
+
+MAVEN_METADATA=https://repo1.maven.org/maven2/com/puppycrawl/tools/checkstyle/maven-metadata.xml
+LATEST_VERSION=$(curl $MAVEN_METADATA | grep -oPm1 "(?<=<latest>)[^<]+")
+echo LATEST_VERSION="$LATEST_VERSION"
+
+MVN_CURRENT_VERSION=$(cat $POM_FILE | grep "^    <version>.*</version>$" | awk -F'[><]' '{print $3}')
+echo MVN_CURRENT_VERSION="$MVN_CURRENT_VERSION"
+
+echo "Checking Version for: "$POM_FILE
+if [ "$MVN_CURRENT_VERSION" != "$LATEST_VERSION"  ]; then
+    cd maven-project
+    mvn -e --no-transfer-progress versions:set -DgroupId=com.github.sevntu-checkstyle -DartifactId=maven-project \
+   -DoldVersion="$MVN_CURRENT_VERSION" -DnewVersion="$LATEST_VERSION-SNAPSHOT"
```

Comment:
respect indentaiton of +2 for line wrap.
```
mvn -e --no-transfer-progress versions:set \
  -DgroupId=com.github.sevntu-checkstyle -DartifactId=maven-project \
  -DoldVersion="$MVN_CURRENT_VERSION" -DnewVersion="$LATEST_VERSION-SNAPSHOT"
```

Reply:
Just for confirmation,
Do I need to update the version or checkstyle version?